### PR TITLE
doc: Clarify cli differences

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Closes https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/469

#### Screenshots

Bildschirmfoto 2021-08-16 um 18 30 17

#### Changes

- Clarify ttn-lw-cli vs tti-lw-cli

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/getting-started/cli/installing-cli/_index.md [hunk 1]
```diff
@@ -8,11 +8,13 @@ This section contains instructions for installing the command-line interface.
 
 <!--more-->
 
+{{% tts %}} CLI is called `ttn-lw-cli`. There is also an enterprise specific CLI called `tti-lw-cli` that is nearly identical, but the enterprise `tti-lw-cli` has additional features for [tenant management]({{< ref "reference/api/tenant" >}}). All other commands are the same. For most users, including Cloud and Community Edition users, `ttn-lw-cli` is sufficient.
+
 ### Package managers (recommended)
 
-{{< tabs/container "Open Source" "Enterprise" >}}
+{{< tabs/container "All Distributions" "Enterprise" >}}
```
Comment (johanstokking) [inline doc/content/getting-started/cli/installing-cli/_index.md:15]: Should we call this then **Cloud and Community** ?

**Review by johanstokking — APPROVED** [review]

**Review by htdvisser — APPROVED** [review]
Nothing to add

File: doc/content/getting-started/cli/installing-cli/_index.md [hunk 2]
```diff
@@ -8,11 +8,13 @@ This section contains instructions for installing the command-line interface.
 
 <!--more-->
 
+{{% tts %}} CLI is called `ttn-lw-cli`. There is also an enterprise specific CLI called `tti-lw-cli` that is nearly identical, but the enterprise `tti-lw-cli` has additional features for [tenant management]({{< ref "reference/api/tenant" >}}). All other commands are the same. For most users, including Cloud and Community Edition users, `ttn-lw-cli` is sufficient.
```
Comment (nejraselimovic) [inline doc/content/getting-started/cli/installing-cli/_index.md:11]: I would rephrase this to something like "There are two versions of TTS CLI: `ttn-lw-cli` and `tti-lw-cli`. For most users `ttn-lw-cli` is sufficient, as it supports a full set of commands to manage all TTS features. For TTS Enterprise, we recommend using `tti-lw-cli`, as this version of the CLI contains additional commands for tenant management, compared to `ttn-lw-cli`."

File: doc/content/getting-started/cli/installing-cli/_index.md (hunk 1 above)
Comment (nejraselimovic) [reply doc/content/getting-started/cli/installing-cli/_index.md:15]: Are "Open Source" and "Enterprise" the "names" of these two CLI versions? Because the linked issue also mentiones "open-source and enterprise client", I'm not sure now what we should do with these tags. Would it be weird if someone using Community installed `tti-lw-cli` and found out that it works?

**Review by neoaggelos — APPROVED** [review]

File: doc/content/getting-started/cli/installing-cli/_index.md (hunk 1 above)
Comment (johanstokking) [reply doc/content/getting-started/cli/installing-cli/_index.md:15]: Hmm. Practically speaking, the CLI is:

- `ttn-lw-cli`: Open Source, Community and Cloud
- `tti-lw-cli`: Enterprise

If we ever were to rename the CLI names, it would be `tts` and `ttse`.

Comment (nejraselimovic) [reply doc/content/getting-started/cli/installing-cli/_index.md:15]: Ok, I would then indeed use "Open Source, Cloud and Community" instead of "All distributions"

**Review by nejraselimovic — APPROVED** [review]
@benolayinka please check the comments above before merging this

**Comment by johanstokking** [comment]
Most notably: https://github.com/TheThingsIndustries/onboarding/pull/81/files
